loan-payment-skip/loan-payment-skip-json-contract.md

```diff
@@ -9,94 +11,105 @@ To get things started, the client will send an initial request structured like s
-| Tracking Record Create Failed. Error: [file maintenance system error message] ||
-| Skip Pay Fee Post Failed. Error=[transaction posting system error message] ||
+| Logging Error Message                                              | Additional Notes (As Needed)                                                                                     |
+| ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------- |
+| No loan types found in CFG Letter file                             | This set of errors represents issues with the configuration letter file setup                                    |
+| Invalid loan tracking type: [4-digit tracking type]                |                                                                                                                  |
+| Invalid sub source code: [3-digit sub source code]                 |                                                                                                                  |
+| Invalid other action: [3-digit other action]                       | Configuration error when OA (other action) CFG file setting is not 0 (share fee comment) or 1 (loan fee comment) |
+| Test mode on but no test account(s) defined                        | Configuration error when TNC (test newest changes) CFG file setting is TRUE, but TML (test member list) is blank |
+| Invalid start/end date(s)                                          |                                                                                                                  |
+| Error Opening Letterfile [config file name]: [sys generated error] |                                                                                                                  |
+| Invalid loan id: [loan id]                                         | This set of errors is for skip payment processing errors                                                         |
+| Attempt to skip a payment on an ineligible loan: [loan id]         |                                                                                                                  |
+| Invalid fee share id: [share id]                                   |                                                                                                                  |
+| Loan Record Update Failed. Error: [file maint system error]        |                                                                                                                  |
+| Tracking Record Create Failed. Error: [file maint system error]    |                                                                                                                  |
+| Skip Pay Fee Post Failed. Error: [tran posting system error]       |                                                                                                                  |
```

Comment:
change this section to somehow show:
Share Skip Pay Fee Post Failed. Error: [tran posting system error]
and
Loan Skip Pay Fee Post Failed. Error: [tran posting system error]